BOS.c

```diff
@@ -2912,71 +2910,69 @@ BOS_Status WriteToRemote(uint8_t module, uint32_t localAddress, uint32_t remoteA
 		switch (format)											
 		{
 			case FMT_BOOL:
-			case FMT_UINT8: 
-				messageParams[2] = *(__IO uint8_t *)localAddress; 
+			case FMT_UINT8:
```

Comment:
@Najibkassab are you sure about this change?

Reply:
it's a mistake, I tested today and it was wrong.
I will fix it and commit the change